content/blog/2016/2016-12-15-declarative-pipeline-beta.adoc

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "Declaring the Beta Release of Declarative Pipeline Syntax "
+tags:
+- pipeline
+- newfeatures
+- blueocean
+author: hrmpw
+---
+
+All the way back in September, Andrew Bayer presented a link:https://www.cloudbees.com/introducing-new-way-define-jenkins-pipelines[sneak peak] of Declarative Pipeline at Jenkins World and there was much rejoicing. Although it has been available in the update center since August, we have waited to recommend it for all of your pipeline needs while we work out the kinks and solidify the syntax. This week we released version 0.71 of the link:https://wiki.jenkins-ci.org/display/JENKINS/Pipeline+Model+Definition+Plugin[_Pipeline-Model-Defintion_] plugin and wanted to crown it as the official Beta version of the Declarative Pipeline Syntax.
+
+=== What is Declarative Pipeline?
+
+We are calling this new syntax Declarative Pipeline to differentiate it from the existing Scripted Pipeline syntax that has always been a part of Pipeline. After listening to many Jenkins users over the last year we felt that, although Scripted Pipeline provides tremendous power, flexibility, and extensibility, the learning curve for a DSL was steep for users new to either Jenkins or Pipeline. Beginning users wanting to take advantage of all the features provided by Pipeline and Jenkinsfiles were forced to learn the DSL or remain limited to the functionality provided by Freestyle jobs.
+
+Declarative Pipeline enables users to connect simple, declarative blocks that define build agents (including Docker), post build actions, environment settings, credentials and all stages that make up the pipeline. It does not replace Pipeline but extends it with a pre-defined structure to lets users focus entirely on the `steps` required at each `stage` without needing to worry about scripting every aspect of the pipeline. Granular flow-control is extremely powerful and Scripted Pipeline syntax will always be part of Pipeline but it's not for everyone.  Best of all, because is is part of Pipeline, all build steps and build wrappers available in Plugins or loaded from Shared Libraries are also available as `steps` in Declarative.
+
+=== Example
+
+[pipeline]
+----
+// Declarative //
```

Comment:
One of the features of the `[pipeline]` block is that it can display both declarative and scripted pipeline.  Could you add a `// Scripted //` section with the same example in scripted form to this block? It would highlight the improvements.